contrib/pg_stat_statements/sql/test_pg_stat_statements.sql

```diff
@@ -0,0 +1,31 @@
+-- start_ignore
+\! gpconfig -c shared_preload_libraries -v 'pg_stat_statements';
+\! gpstop -raiq;
```

Comment:
the '-i' parameter is deprecated. is there any reason to use it?

Reply:
Only reason is that it was done so in sample code (tests in auto_explain). Will change to avoid using deprecated parameters.